flow-typed/platform.js

```diff
@@ -16,6 +16,24 @@ declare type MWPState = {
 		enableServiceWorker: boolean,
 		requestLanguage: string,
 		supportedLangs: Array<string>,
+		member: { 
+			id: number,
+			status: number,
+			timestamp: number,
+			bs: number,
+			// time zone
+			tz: string,
+			zip: number,
+			country: string,
+			city: string,
+			state: string,
+			lat: number,
+			lon: number,
+			ql: boolean,
+			scope: string,
+			rem: number,
+			org: number, 
```

Comment:
can this be any number, or is it an enum like `0 | 1 | 2` or something?

Reply:
its 1 or its not there